addons/point_of_sale/tools/posbox/configuration/connect_to_server_wifi.sh

```diff
@@ -58,67 +58,92 @@ function connect () {
 	# make network choice persistent
 	if [ -n "${ESSID}" ] ; then
 		if [ -n "${PERSIST}" ] ; then
-			logger -t posbox_connect_to_wifi "Making network selection permanent"
+			logger -t posbox_connect_to_server_wifi "Making network selection permanent"
```

Comment:
Like your previous PR you can make only one file (remove connect_to_server_wifi.sh)

Reply:
To keep focused on this bug, better to do this when a conflict will appears with the forward bot, no?

Reply:
ok